client/helpers/get-key-value-pairs.js

```diff
@@ -18,6 +18,11 @@ const getKeyValuePairs = (event) => {
           routeLink: value.routeLink,
           value: value.text,
         });
+      } else if (value.duration !== undefined) {
+        kvps.push({
+          key,
+          value: moment.duration(value.duration, 'seconds').format(),
```

Comment:
It will properly format timeout value, right? For `300, 'seconds'` will it show `5m`?

Reply:
yes, that will render as 5m